sections/summary.tex

```diff
@@ -53,5 +53,8 @@ \subsection{Collaboration}
 Finally, the models were trained using arbitrary settings for the batch sizes, hyperparameters, learning and dropout rates.
 
 
-\subsection{Conclusion/ending}
-% TODO: Conclusion goes here
\ No newline at end of file
+\subsection{Conclusion}
+Having used weather data gathered from different weather sensors we have trained and compared different machine learning models for time series forecasting.
+
+The focus of the paper was to show that the transformer machine learning model would outperform previous state-of-the-art statistical and machine learning techniques. This we have shown to be the case. 
+Additionally, we developed a web application to show the results of the trained models and the user is able to interact with this web application by using a dropdown to select a model and a dataset. This will then make temperature predictions.
```

Comment:
```suggestion
Additionally, we developed a web application to show the results of the trained models, and the user is able to interact with this web application by using a dropdown to select a model and a dataset. This will then show the temperature predictions.
```